src/clj_kondo/impl/linters.clj

```diff
@@ -191,6 +191,21 @@
                    (str ns-name "/" fn-name)
                    (some? const-true))))))))
 
+(defn- lint-missing-clause-in-try
+  [call]
+  (when (utils/one-of (:name call) [try try+])
```

Comment:
Should we include [slingshot's](https://github.com/scgilardi/slingshot) version `try+` for linting? I added it for now cause it seems that version is widely used too.

Reply:
I'd prefer if we left it out for now. People can use `:lint-as` if they want to lint `clojure.core/try` as `foo.bar/try+`.

Reply:
Sure, removed it.